src/graph/backend/dnnl/fusion_info.cpp

```diff
@@ -73,8 +73,9 @@ dnnl::primitive_attr make_dnnl_primitive_attr(
         for (const auto &in_scales : fusion_info.input_scales_) {
             size_t in_scales_indices = in_scales.first;
             const op_t *in_scales_op = in_scales.second->get_op();
-            assertm(fusion_info.with_runtime_scales(true, in_scales_indices),
-                    "only support runtime src scales.\n");
+            VCHECK_FUSION_INFO(
+                    fusion_info.with_runtime_scales(true, in_scales_indices),
+                    attr, "primitive only supports runtime src scales.");
```

Comment:
nit: is it possible to have the info of which primitive? it's not very helpful to just know there's a primitive out of all the primitives that only supports runtime src scales..

Reply:
Thanks for your comments. add the op info in the log. could you take another look?